examples/forms.amp.html

```diff
@@ -72,7 +86,8 @@
 <form method="post"
       action="/form/html/post"
       action-xhr="/form/echo-json/post"
-      target="_blank">
+      target="_blank"
+      on="submit:lightbox1">
```

Comment:
Correct. Although needed to be tested. You can also call `actionService.trigger()` from your existing global submit handler. Or even from both: global and the one in the `AmpForm`. As long as one even is always sent and no more than one. But I'd generally gravitate toward the global handlers, either via `addEvent` or global submit handler.